strings/tips.txt

```diff
@@ -0,0 +1,205 @@
+As a Revenant, the illness inflicted on humans by Blight can be easily cured by lying down or with holy water, making it best used on targets that have no time to lie down, such as humans in combat.
+As a Revenant, fastmos is your friend. Breaking windows to space can cause massive damage and mayhem.
+As a Swarmer, you can deconstruct more things than you think. Try deconstructing light switches, buttons, air alarms and more. Experiment!
+As a Swarmer, you can teleport fellow swarmers away if you think they are in danger.
+As a Ghost, you can double click on just about anything to follow it. Or just warp around!
+As a Devil, you gain power for every three souls you control, however you also become more obvious.
+As a Devil, as long as you control at least one other soul, you will automatically resurrect, as long as a banishment ritual is not performed.
+At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth. 
+As a Security Officer, remember that correlation does not equal causation. Someone may have just been at the wrong place at the wrong time!
+As a Security Officer, remember that you can attach a sec-lite to your taser or your helmet!
+As a Terror Spider, leave wrapping of corpses to green spiders unless absolutely necessary. They need it to lay eggs!
+As a Prince of Terror, you are powerful but usually on your own. Consider hit and run tactics to regenerate your health between engagements.
+As a Brown Terror Spider, focus on smashing welded vents. You are weak in direct combat, but you can enable your comrades to go almost anywhere.
+As a Terror Spider, protect the queen at all cost!
+As a Terror Spider, don't forget you can move your nest if the crew is pressuring you too badly.
+As a Terror Spider, try putting webs on white floor tiles, they are much harder to see there.
+As a Terror Spider, sometimes quickly pulling an enemy into a web can ensure victory in melee combat.
+You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
+When hacking doors, cutting and mending a "test light wire" will restore power to the door.
+When crafting most items, you can either manually combine parts or use the crafting menu.
+Suit storage units not only remove blood and dirt from clothing, but also radiation!
+Sleeping in a bed can heal minor amounts of brute and burn damage.
+The Experimentor has a chance to spawn hostile mobs. Be ready to run for your life when using it.
+Space ruins can contain all kinds of interesting goodies, even a wand that can raise the dead!
+Having a mindshield, being a human and having an agent ID card all make officer Beepsky consider you less arrest-worthy. What a racist bot.
+You can make lasertag turrets, for the ultimate lasertag tournament.
+Blob structures take half damage from brute damage. Use lasers.
+You can hide paper in vents, you have to use a screwdriver to open it first however.
```

Comment:
You can hide paper in vents, **but** you have to use a screwdriver to open it first ~however~.

Reply:
fixed in next commit